docs/data-sources/apig_api_basic_configurations.md

```diff
@@ -0,0 +1,119 @@
+---
+subcategory: "API Gateway (Dedicated APIG)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_apig_api_basic_configurations"
+description: |-
+  Use this data source to get the basic configuration list of the APIs within HuaweiCloud.
+---
+
+# huaweicloud_apig_api_basic_configurations
+
+Use this data source to get the basic configuration list of the APIs within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "instance_id" {}
+
+data "huaweicloud_apig_api_basic_configurations" "test" {
+  instance_id = var.instance_id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the data source.
+  If omitted, the provider-level region will be used.
+
+* `instance_id` - (Required, String) Specifies the ID of the dedicated instance to which the APIs belong.
+
+* `api_id` - (Optional, String) Specifies the ID of the API.
+
+* `name` - (Optional, String) Specifies the name of the API. Fuzzy search is supported.
+
+* `group_id` - (Optional, String) Specifies the ID of the API group to which the APIs belong.
+
+* `type` - (Optional, String) Specifies the type of the API.  
+  The valid values are `Public` and `Private`.
+
+* `request_method` - (Optional, String) Specifies the request method of the API.  
+  The valid values are `GET`, `POST`, `PUT`, `DELETE`, `HEAD`, `PATCH`, `OPTIONS` and `ANY`.
+
+* `request_path` - (Optional, String) Specifies the request address of the API. Fuzzy search is supported.
+
+* `request_protocol` - (Optional, String) Specifies the request protocol of the API.  
+  The valid values are `HTTP`, `HTTPS`, `BOTH` and `GRPCS`.
+
+* `security_authentication` - (Optional, String) Specifies the security authentication mode of the API request.  
+  The valid values are `NONE`, `APP`, `IAM` and `AUTHORIZER`.
+
+* `vpc_channel_name` - (Optional, String) Specifies the name of the VPC channel. Fuzzy search is supported.
+
+* `precise_search` - (Optional, String) Specifies the parameter name for exact matching.  
+  The valid value is `name` or `req_uri`.
```

Comment:
```suggestion
* `precise_search` - (Optional, String) Specifies the parameter name that you want to match exactly.
  The valid values are:
  + **name**: API name, corresponding to the field `name` in this data source arguments.
  + **req_uri**: Request path, corresponding to the field `request_path` in this data source arguments.

  This parameter takes effect only after the corresponding parameter(s) is(are) set.
```
And check whether this parameter can be set to multiple enumerated values, e.g. `name,req_uri`.

Reply:
fixed